app/launcher.ts

```diff
@@ -0,0 +1,206 @@
+/* eslint-disable no-invalid-this */
+/* eslint-disable @typescript-eslint/naming-convention */
+import { IProgram } from "gatsby/internal"
+import { readJSON } from "fs-extra"
+import {
+  fork,
+  ChildProcessPromise as ChildProcess,
+  Output,
+} from "promisify-child-process"
+import type { PackageJson } from "gatsby"
+import path from "path"
+import detectPort from "detect-port"
+import fixPath from "fix-path"
+import {
+  LogObject,
+  StructuredEventType,
+  GlobalStatus,
+  Status,
+  IPCMessageType,
+  IPCMessage,
+} from "./ipc-types"
+
+fixPath()
```

Comment:
I don't like this module level side-effect. What does this do? Can it be wrapped in initialization function?